packages/metal-component/src/Component.js

```diff
@@ -276,6 +282,16 @@ class Component extends EventEmitter {
 	}
 
 	/**
+	 * Forces a an update that ignores `shouldUpdate` for components whose
+	 *     render depends on external variables.
+	 */
+	forceUpdate() {
```

Comment:
As per React, [forceUpdate takes a callback](https://reactjs.org/docs/react-component.html#forceupdate)... probably because it works asynchronously... should we take one as well? Do we have usage examples of this?

Reply:
It won't hurt to add the functionality, though technically someone could achieve this by listening to the next `rendered` event.

```javascript
component.once('rendered', function() {
  console.log('just force updated');
});
component.forceUpdate();
```

Though I think we should add the callback even if it's just for cleanliness.

```javascript
component.forceUpdate(function() {
  console.log('just force updated');
});
```